Sources/StreamVideo/HTTPClient/InternetConnection.swift

```diff
@@ -26,7 +26,7 @@ extension Notification {
 ///
 /// Basically, it's a wrapper over legacy monitor based on `Reachability` (iOS 11 only)
 /// and default monitor based on `Network`.`NWPathMonitor` (iOS 12+).
-class InternetConnection: @unchecked Sendable {
+public final class InternetConnection: @unchecked Sendable {
```

Comment:
Making Internet connection public so it can be use from SwiftUI SDK. Let me know your thoughts